skfp/fingerprints/map.py

```diff
@@ -48,10 +47,11 @@ class MAPFingerprint(BaseFingerprintTransformer):
         Number of iterations performed, i.e. maximum radius of resulting subgraphs.
         Another common notation uses diameter, therefore MAP4 has radius 2.
 
-    variant : {"raw_hashes", "bit", "count"}, default="bit"
-        Which variant to fingerprint to use. ``"raw_hashes"`` returns MinHash values.
-        ``"bit"`` folds values into binary vector, and ``"count"`` uses folding with
-        counting.
+    include_duplicated_shingles : bool, default=False
+        Whether to include duplicated shingles in the final fingerprint.
+
+    counts : bool, default=False
```

Comment:
`count`, not `counts`, to follow other fingerprints

Reply:
Done